wandb/sdk/launch/builder/build.py

```diff
@@ -92,7 +92,8 @@ def get_docker_user(launch_project: LaunchProject, runner_type: str) -> Tuple[st
 CUDA_SETUP_TEMPLATE = """
 FROM {cuda_base_image} as base
 # TODO: once NVIDIA their linux repository keys for all docker images
-RUN apt-key adv --fetch-keys http://developer.download.nvidia.com/compute/cuda/repos/$(cat /etc/os-release | grep ^ID= |  cut -d "=" -f2 )$(cat /etc/os-release | grep ^VERSION_ID= |  cut -d "=" -f2 | sed -e 's/[\".]//g' )/$(uname -i)/3bf863cc.pub
+RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/$(cat /etc/os-release | grep ^ID= |  cut -d "=" -f2 )$(cat /etc/os-release | grep ^VERSION_ID= |  cut -d "=" -f2 | sed -e 's/[\".]//g' )/$(uname -i)/3bf863cc.pub
```

Comment:
@KyleGoyette the `diff` for `3bf863cc.pub` with 1804 and 2004 is null, so do you really need this bash magic?